src/main/java/com/enonic/xp/app/welcome/WelcomePageScriptBean.java

```diff
@@ -142,8 +141,7 @@ public String getContentStudioUrl()
         {
             if ( application.getKey().equals( ApplicationKey.from( "com.enonic.app.contentstudio" ) ) )
             {
-                return ServletRequestUrlHelper.createUri( ServletRequestHolder.getRequest(),
-                                                          "/admin/" + application.getKey() + "/main" );
+                return ServletRequestUrlHelper.createUri( ServletRequestHolder.getRequest(), "/admin/" + application.getKey() + "/main" );
```

Comment:
What is the difference? 
If none -  exclude it from change is it is possible to cherry pick to 7.x

Reply:
This is auto-format and we are supposed to stick with it, are we not ?
Besides there are many more places it affected in this PR

Reply:
Revert all auto-format, please.
We want this pool cherry-picked to another branch

Reply:
😒